### PR TITLE
How to add platform administrator

### PR DESCRIPTION
**Pull request by Erwyn**

### CODE REVIEW

**Review by LeSuisse — COMMENTED** [review]
Does this need to be another category?

Should not this go under Administration guide >> Application management?

**Comment by Erwyn** [comment]
> Should not this go under Administration guide >> Application management?

We wanted to push to try on some documentation organization following the [diataxis](https://diataxis.fr) framework as was once suggested by @vaceletm. The idea of this section is straight to the goal how to without any conceptul explaination of anything whatsoever. For instance we wont discuss what permission delegation is here.

**Comment by LeSuisse** [comment]
Hum okay. Are we sure this is right approach to introduce this paradigm shift?

Without re-organizing what we currently have I'm afraid we are going to end up with a big bag of stuff into this section with even less guidelines than today.

I'm not against it but having some context about why the documentation is organized in a specific way would be helpful next time I want to add content to it. An ADR would be nice.

**Comment by Erwyn** [comment]
> Hum okay. Are we sure this is right approach to introduce this paradigm shift?

No, that's why we don't want to reorganize everything for now but test the idea. Moreover, it's not only an organization shift but also content change that we won't be able to tackle in a one-shot. For instance, one could argue that in the Diataxis prism our current document is a mixed bag of *Explaination* and *Reference*. We wanted to start with introducing those how-to's first as it is what we identify as missing the most, Diataxis or not, for new users.

> Without re-organizing what we currently have I'm afraid we are going to end up with a big bag of stuff into this section with even less guidelines than today.

How-to's are supposed to be kind of a bag of stuff. It is very directive and in no way an explanation of any of the underlying mechanisms, very much different from what is usually produced as documentation from the development team.

> I'm not against it but having some context about why the documentation is organized in a specific way would be helpful next time I want to add content to it. An ADR would be nice.

I can add an ADR if you feel like it, at least explaining what how-to's are. I can mention what Diataxis is but I'm not sure I want to push more as it is still something exploratory here.

File: languages/en/how-to/administration/add-a-user-as-platform-administrator.rst [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+.. image:: /images/how-to/administration/permission-delegation-platform-administrator.png
+   :alt: permission delegation interface
+
+1. As a platform administrator, click the two red wheels icon at the top right of your screen.
+2. Go to the ``Permission delegation`` entry in the left sidebar, under the ``Users`` section.
+3. Check if you already have an ``Admin`` group. If not:
+    1. Create one by clicking on the ``Add group`` button.
+    2. Add the ``Platform administration`` permission by clicking the ``Add permissions`` button.
+4. On the right part of the screen, use the ``Add user`` field to add one of your users in the ``Admin`` group.
```
Comment (LeSuisse) [inline languages/en/how-to/administration/add-a-user-as-platform-administrator.rst:12]:
```suggestion
4. On the right part of the screen, use the ``Add user`` field to add one of your users to the ``Admin`` group.
```

Not sure but the `in` feels weird to me.

File: languages/en/index.rst [hunk 2]
```diff
@@ -19,3 +19,4 @@ You can learn more about the project on `Tuleap web site <https://www.tuleap.org
    admin
    deploy
    dev
+   how-to
```
Comment (LeSuisse) [inline languages/en/index.rst:22]: This should probably be the first category in the menu instead of the last after the obscure (for most) dev guide.

**Review by LeSuisse — COMMENTED** [review]
I'm fine with experimenting but without setting determining when the it is successful or not it is hard to know when to stop or when to continue.

I understand that re-organizing everything at once is not possible. Trying to re-work the inside of either the user or the administration guide would make more sense to me instead of adding another section which we will be subdivided.

Anyway I understand this has likely been discussed elsewhere and I do not care enough about it to continue to discuss this in those circumstances.

**Pull request merged**